README.md

```diff
@@ -29,3 +29,31 @@ nuked.render({
   type: 'table',
 });
 ```
+
+Look into [Build a simple mashup using nebula.js](https://qlik.dev/tutorials/build-a-simple-mashup-using-nebulajs) to learn more
+
+## Visualization Extensions Usage
+
+### Steps to create a sn-table extension as as a Qlik Sense extension
+
+Install all dependencies:
+
+```sh
+yarn
+```
+
+Build nebula.js visualization
+
+```sh
+yarn build
+```
+
+Build a nebula visualization as a Qlik Sense extension
+
+```sh
+yarn sense
+```
+
+| [Saas Edition of Qlik Sense](https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Admin/mc-extensions.htm) | [Qlik Sense Enterprise](https://help.qlik.com/en-US/sense-developer/May2021/Subsystems/Extensions/Content/Sense_Extensions/Howtos/deploy-extensions.htm) | [Qlik Sense Desktop](https://help.qlik.com/en-US/sense-developer/May2021/Subsystems/Extensions/Content/Sense_Extensions/Howtos/deploy-extensions.htm) |
```

Comment:
the Qlik Sense Desktop heading looks right-centered for some reason

Reply:
Fixed